plover/gui_qt/main_window.py

```diff
@@ -143,6 +143,7 @@ def __init__(self, engine, use_qt_notifications):
             self.on_configure()
         # Apply configuration settings.
         config = self._engine.config
+        self._hidden_to_tray_message_shown = config['start_minimized']
```

Comment:
Given what this do, the variable name is somewhat misleading. I'm thinking "[hidden to tray] message show next time" but that's quite long.